core/src/hlist.rs

```diff
@@ -621,6 +621,29 @@ macro_rules! gen_inherent_methods {
             {
                 HFoldRightable::foldr(self, folder, init)
             }
+
+            /// Extend the contents of this HList with another HList
```

Comment:
Sorry, it's been a while, so I forgot about this, but there's an implementation of `Add` that looks identical: can we re-implement this method in terms of that?

https://github.com/lloydmeta/frunk/blob/341df3e921fee3e5891724abf4906869fa18284f/core/src/hlist.rs#L762-L786

Reply:
Oh... I didn't even see this impl; it indeed does the exact same thing as the `Extend` trait. Do you still want an inherent `extend` method? It might help with the discoverability problem I just had lol. Or adding something to the `hlist` module docs about the `Add` impl might be enough too.

Reply:
> Oh... I didn't even see this impl; it indeed does the exact same thing as the `Extend` trait. Do you still want an inherent `extend` method? It might help with the discoverability problem I just had lol. Or adding something to the `hlist` module docs about the `Add` impl might be enough too.

I would love to keep the inherent `extend` method. It 100% helps with discoverability.